docs/en/event-daemon/event-daemon-plugins.md

```diff
@@ -4,3 +4,201 @@ title: Plugins
 pagename: event-daemon-plugins
 lang: en
 ---
+
```

Comment:
As mentioned in the example-plugins doc, this content needs to be in example-plugins, and the content currently there needs to be in this document.